src/main/java/com/teragrep/pth10/steps/teragrep/TeragrepBloomStep.java

```diff
@@ -100,7 +102,7 @@ public TeragrepBloomStep(
         this.outputCol = outputCol;
         this.estimateCol = estimateCol;
 
-        if (mode == BloomMode.ESTIMATE) {
+        if (mode == BloomMode.ESTIMATE || mode == BloomMode.AGGREGATE) {
```

Comment:
In-constructor code is not ideal, but as far as I remember there is an issue in StepList where properties cannot be added in any other place. Perhaps a new issue?